baker/algolia/configureAlgolia.ts

```diff
@@ -77,15 +81,20 @@ export const configureAlgolia = async () => {
         ...baseSettings,
         searchableAttributes: [
             "unordered(title)",
+            "unordered(excerpt)",
+            "unordered(tags)",
```

Comment:
just wondering what is the reason for renaming `_tags`? Is it just cosmetic and nothing to do with some features of the reserved `_tags` attribute name you wanted to disable? AFAIK the only feature is automatic `filterOnly(_tags)` in `attributeForFacetings`, but I'm not sure how we use those at the moment anyway (cf. other comment below)

Reply:
Ah thank you, I was totally unaware that [`_tags` has a special meaning in Algolia](https://www.algolia.com/doc/guides/managing-results/refine-results/filtering/how-to/filter-by-tags/#the-difference-between-_tags-and-a-custom-array).
To me, it was just the cosmetics of `_tags`.
However, since we don't currently use filtering or faceting - and can still do that even with it not being called `_tags` - I would be inclined to keep it as `tags`. Do you agree?

Reply:
yep that works for me. Also, it might actually be better to have it as a non-magic attribute so we can reason with all attributes in the same way.